packages/connect/src/utils/pathUtils.ts

```diff
@@ -195,6 +195,7 @@ export const fixPath = <T extends PROTO.TxInputType | PROTO.TxOutputType>(utxo:
     // This is only a part of API wide issue: https://github.com/trezor/trezor-suite/issues/4875
     // it works only in runtime. type T needs to have address_n as string, but currently we are using Protobuf declaration
     if (utxo.address_n && typeof utxo.address_n === 'string') {
+        // @ts-expect-error Related to previous comment
```

Comment:
Can you verify @mroz22 
<img width="782" alt="Screenshot 2022-11-16 at 14 50 41" src="https://user-images.githubusercontent.com/5837757/202198344-67183456-2846-450a-a221-796ac531f66d.png">

It seems it could be related to comment above

Reply:
I see `Unused '@ts-expect-error' directive.` and type-check pass fine for me locally without the ts-expect-error 🤷 
![image](https://user-images.githubusercontent.com/3729633/202475589-3d580b96-8b5b-4601-8acb-827beb7e2426.png)